cmd/validate/ruby/ruby_validator.go

```diff
@@ -0,0 +1,54 @@
+package rubyvalidate
+
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	peVersion 		string
+	puppetDev 		string
+	puppetVersion string
+)
+
+type RubyValidator struct{}
+type command strings.Builder
+
+func (r *RubyValidator) SetCommand(command string, options []string) (err error) {
+	// Check binary exists in Ruby gems path - should always exist in Docker land???
+	commandFilePath := filepath.Join(os.Getenv("RUBY_BIN_CACHE"), command)
+	_, err = os.Stat(commandFilePath)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (r *RubyValidator) ValidInContext() (valid bool) {
+	return true
+}
+
+func CreateValidateRubyCommand() *cobra.Command {
+	tmp := &cobra.Command{
+		Use: 		"ruby [flags]",
+		Short:	"Run Ruby code validators",
+		Long: 	"Run Ruby code validators",
+		RunE:		executeValidateRuby,
+	}
+
+	tmp.Flags().StringVar(&peVersion, "pe-version", "", "Puppet Enterprise version to run tests or validations against.")
+	tmp.Flags().StringVar(&puppetDev, "puppet-dev", "", "When specified, PDK will validate or test against the current Puppet source from github.com. To use this option, you must have network access to https://github.com")
+	tmp.Flags().StringVar(&puppetVersion, "puppet-version", "", "Puppet version to run tests or validations against")
```

Comment:
`puppet-version` and potentially `pe-version` will not be configured this way anymore, I'm thinking @da-ar ?